polyfill/lib/plaindate.mjs

```diff
@@ -71,6 +71,10 @@ export class PlainDate {
     if (!ES.IsTemporalDate(this)) throw new TypeError('invalid receiver');
     return GetSlot(this, CALENDAR).era(this);
   }
+  get eraYear() {
+    if (!ES.IsTemporalDate(this)) throw new TypeError('invalid receiver');
+    return GetSlot(this, CALENDAR).eraYear(this);
```

Comment:
Later in this one, mentioned it in a TODO above. It will use the values from [this table](https://github.com/tc39/proposal-temporal/pull/1319/files#diff-3a221d4b1eb72b9134f63736e30ed2d2c11eccd7ee2fb34beb3c9ac8e1c3247cR1491), so should be nothing unexpected; I wanted to get out the table & abstract op before today's meeting at least.